src/components/timecard/edit-shift-hours/EditShiftHours.spec.jsx

```diff
@@ -9,35 +10,42 @@ const newTimeEntry = {
 };
 
 const timecardService = require('../../../api/services/timecardService');
-const mockSaveTimeEntry = jest.spyOn(timecardService, 'saveTimeEntry');
+const mockCreateTimeEntry = jest.spyOn(timecardService, 'createTimeEntry');
+const mockUpdateTimeEntry = jest.spyOn(timecardService, 'updateTimeEntry');
 
 describe('EditShiftHours', () => {
-  it('should call saveTimeEntry when pressing save with no existing time entry', async () => {
+  it('should call createTimeEntry when pressing save with no existing time entry', async () => {
+    let timecardDate = '2022-09-01';
+    let inputtedStartTime = '08:00';
     renderWithTimecardContext(
       <EditShiftHours
         setShowEditShiftHours={jest.fn()}
         timeEntry={newTimeEntry}
         index={0}
       />,
       {
-        timecardDate: '2022-09-01',
+        timecardDate: timecardDate,
       }
     );
 
     act(() => {
       const startTimeInput = screen.getByTestId('shift-start-time');
-      fireEvent.change(startTimeInput, { target: { value: '08:00' } });
+      fireEvent.change(startTimeInput, {
+        target: { value: inputtedStartTime },
+      });
 
       const saveButton = screen.getByText('Save');
       fireEvent.click(saveButton);
     });
 
     await waitFor(() => {
-      expect(mockSaveTimeEntry).toHaveBeenCalledWith(
+      expect(mockCreateTimeEntry).toHaveBeenCalledWith(
         {
           ownerId: 1,
           timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
-          actualStartTime: '2022-09-01T08:00:00+00:00',
+          actualStartTime: formatDateTimeISO(
+            timecardDate + ' ' + inputtedStartTime
+          ),
```

Comment:
I think it would be valuable to have the expected `actualStartTime` as a string, so then if the `formatDateTimeISO` function changes, this test would need changing too

Reply:
So the reason I had to change it to expect it as the iso format is because of the timezone offset. If ran on a sever in a GMT timezone it will pass with `2022-09-01T08:00:00+00:00` but in BST it would fail as it's expecting +1 offset.

It's a good point, so I guess the question is are we saying we want to enforce that UI tests are always ran in UTC/GMT on **any** environment? If so, then yeah it can be reverted.
Or don't enforce a a timezone for it to be ran in and include the full iso formatted date in these types of tests?

Welcome anyone else' thoughts too!

Reply:
Have now updated after further discussion in slack.